test/functional/legacy/packadd_spec.lua

```diff
@@ -334,6 +334,8 @@ describe('packadd', function()
         let g:sequence = ''
         runtime! ALL extra/bar.vim
         call assert_equal('runstartopt', g:sequence)
+
+        call delete(s:topdir, 'rf')
```

Comment:
This should be executed by TearDown() in `after_each` but I don't mind 1 more call.